crates/biome_cli/src/metrics.rs

```diff
@@ -353,7 +353,7 @@ mod tests {
             assert_eq!(histogram.mean(), mean);
 
             let sum = (1..=5).fold(0.0, |sum, i| {
-                let sample = i as f64 * scale;
```

Comment:
This rule feels too restrictive to me. What's the reason for including it?

Reply:
[cast_lossless](https://rust-lang.github.io/rust-clippy/master/#/cast_lossless?levels=allow,none&groups=pedantic,restriction) avoids unsafe cast. However, in this particular case, I accept it is too pedantic. I will disable the rule.